mathesar_ui/.eslintrc.cjs

```diff
@@ -73,6 +73,8 @@ module.exports = {
         'no-constant-condition': 'off',
         '@typescript-eslint/restrict-template-expressions': 'off',
         '@typescript-eslint/no-unsafe-argument': 'off',
+        'no-sequences': 'off',
```

Comment:
Why do we have this rule turned off? I'm okay without it, but I'm curious of the case.

Reply:
Good question. Looks like I actually removed the code that I needed this for. So we could turn the rule back on if we like. But it can be nice sometimes. I was using it [here](https://github.com/centerofci/mathesar/commit/573fedab0e62c51fb0885915b68401244abb599b#diff-10328dc2a93d1d4551bbc3b12584b2458c42fd43b0af08ce79a78b2fb7ea026fR38) for a reactive statement like this:

```ts
$: table, init();
```

That code reads like this: _"when `table` changes, re-initialize the form."_ I've seen other people in the Svelte community embrace this pattern, even though it's a bit weird.

I later ended up building the form system which handles this logic much more elegantly.